domain/model/mesh_results.py

```diff
@@ -128,7 +143,8 @@ def filter(self, from_agent, to_agent: AgentID, metric: MetricType) -> List[Tupl
         if metric == MetricType.PACKET_LOSS:
             return [(i.time, i.packet_loss_percent) for i in items]
 
-        return []
+        logger.debug("MeshResults.filter: unknown metric type")
+        return [(None, None)]
```

Comment:
This pretends that there was a single filter result item, while there actually was 0 items.